beacon-chain/sync/rpc_beacon_blocks_by_range.go

```diff
@@ -124,6 +126,25 @@ func (r *Service) writeBlockRangeToStream(ctx context.Context, startSlot, endSlo
 		traceutil.AnnotateError(span, err)
 		return err
 	}
+	// handle genesis case
+	if startSlot == 0 {
```

Comment:
Nit pick, refactor this into its own function

Reply:
Ok done